README.md

```diff
@@ -1,5 +1,92 @@
-# RS/XP 2023
+![image](https://user-images.githubusercontent.com/50504050/228393842-d5ebe930-d3ae-4fc8-a18a-f1d91a2b98e2.png)
 
-Repositório contendo o código do app do RS/XP 2023.
+# Rocketseat Experience 2023 - Mobile App
+Repositório do app construido pela comununidade para o [maior evento de programação do Brasil](https://www.rocketseat.com.br/eventos/rsxp)! :brazil:
+
+## Links rápidos ↗
+
+- [Documentação Principal | Notion 📖](https://rsxp-2023.notion.site/App-Mobile-d98e3a215a604a8192b8aa99d7887708)
+- [Layout | Figma 🎨](https://www.figma.com/file/mSZqH7CG5fXBO0jsHMyMts/RS%2FXP-23-App?node-id=0%3A1&t=VZ191dsu8YEtsdFX-1)
+
+
+## Tech Stack 💜
+A *stack* foi escolhida pela comunidade durante a [live de kick-off](https://www.youtube.com/watch?v=dDO0YKuHABY).
+
+**📱 Mobile:**
+- [Expo](https://github.com/expo/expo)
+- [TypeScript](https://github.com/microsoft/TypeScript)
+- [NativeWind](https://github.com/marklawlor/nativewind) *(Tailwind para o React Native)*
+
+**🏧 Server:**
+- [NestJS](https://github.com/nestjs/nest) `(REST)`
+- [Prisma](https://github.com/prisma/prisma)
+- [PostgreSQL](https://github.com/postgres) [`Neon Database`](http://neon.tech/)
+- [TypeScript](https://github.com/microsoft/TypeScript)
+- [Sympla API](https://www.sympla.com.br/api-doc/index.html)
+
+
+## Funcionalidades 🚀
+
+Até o momento, levando em consideração o prazo de 2 semanas *(Due: 11/04/2023)*, listamos algumas funcionalidades básicas que devemos entregar. Caso sobre tempo, podemos trabalhar em ideias *backlog*.
+
+- [ ]  **Comprar ingresso**
```

Comment:
Parabéns pela iniciativa, ficou muito bom esse novo README.
Oque acha de separar as funcionalidades de acordo com o figma?

Uma sugestão para essa abordagem:

### Funcionalidades 1.0 (MVP):

- [ ] **Comprar ingresso**
Redirecionar o usuário para o [Sympla](https://www.sympla.com.br/) para efetuar a compra.

- [ ]  **Visualizar meu ingresso**
Uma vez que tenha o código localizador do ingresso emitido pelo Sympla, o usuário poderá conectar seu ingresso à sua conta no app e, dessa forma, terá acesso pelo app.

- [ ]  **Compartilhar ingresso nas redes sociais**
Como uma forma de hypar o evento, o usuário poderá gerar uma imagem - *thumbnail* com sua foto para divulgar nas redes! `#VouNaRSXP! 🚀💜`

### Funcionalidades 2.0:

- [ ]  **Autenticação**
- [ ]  **Sistema de convites**
    - Desconto p/ convidar
    - brindes p/ quem mais convidar
- [ ]  **Countdown pro evento**
- [ ]  **Dados gerais do evento**
    - Localização
    - Data & Hora
- [ ]  **FAQ** - (*Frequently Asked Questions / Perguntas Frequentes*)

- [ ]  **Feed de notícias**
Criar algo tipo um Twitter interno para evento. Tal aba será dedicada aos palestrantes ou organizadores para enviar avisos para os membros.

- [ ]  **Lista dos palestrantes**

Reply:
@FelipeQueiroz Concordo! Irei modificar com essas sugestões!